yjit/src/codegen.rs

```diff
@@ -5012,8 +5128,13 @@ fn gen_send_iseq(
         return CantCompile;
     }
 
+    if kw_arg_num != 0 && flags & VM_CALL_ARGS_SPLAT != 0 {
+        gen_counter_incr!(asm, send_cfunc_splat_with_kw);
+        return CantCompile;
+    }
+
     if doing_kw_call && flags & VM_CALL_ARGS_SPLAT != 0 {
-        gen_counter_incr!(asm, send_iseq_splat_with_kw);
+        gen_counter_incr!(asm, send_cfunc_splat_with_kw);
```

Comment:
This is in `send_iseq`, but we're using cfunc counters.

Reply:
Will fix